contrib/examples/apiserver/instance.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: test-instance
   namespace: test-ns
 spec:
-  externalClusterServiceClassName: test-serviceclass
-  externalClusterServicePlanName: example-plan-1
+  clusterServiceClassName: test-serviceclass
```

Comment:
Wonder if there's way to find a way to get rid of the cluster* prefix. At first I thought, great because it's in the cluster service instance, but it's not. if there was a way to get the context 'for free' as being cluster or namespaced, would be nice to not have to specify it here.

Reply:
I suspect we may allow for the same name to be used at both scopes, which mean we'll need a way for the user to tell us which one they mean.

Reply:
Yeah, I think we need to keep `cluster` prefix for the forward compatibility with namespace-scoped counterparts in future.

Reply:
@nilebox is correct; let's not do this precise rename